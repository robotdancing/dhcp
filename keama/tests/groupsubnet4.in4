# Group with DHCPv4 subnet declaration config

# authoritative is mandatory
authoritative;

# parameter which will be changed in subnet
default-lease-time 1200;

# group declaration
group foobar {
    # option
    option domain-search "example.com", "example.org";

    # parameters
    default-lease-time 3600;
    ignore-client-uids false;

    # DHCPv4 subnet declaration
    subnet 10.5.5.0 netmask 255.255.255.224 {
        # at least one pool is required
        pool {
            range 10.5.5.5 10.5.5.10;
        }
	interface "bar";
        default-lease-time 1800;	
    }
}
