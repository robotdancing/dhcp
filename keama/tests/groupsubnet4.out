{
  # Group with DHCPv4 subnet declaration config
  # authoritative is mandatory
  "Dhcp4": {
    "valid-lifetime": 1200,
    "interfaces-config": {
      "interfaces": [
        "bar"
      ]
    },
    "subnet4": [
      # DHCPv4 subnet declaration
      {
        "subnet": "10.5.5.0/27",
        "pools": [
          # at least one pool is required
          {
            "pool": "10.5.5.5 - 10.5.5.10"
          }
        ],
        "interface": "bar",
        "valid-lifetime": 1800,
        "option-data": [
          # option
          {
            "space": "dhcp4",
            "name": "domain-search",
            "code": 119,
//          "original-data": "\"example.com\", \"example.org\"",
            "data": "example.com, example.org"
          }
        ],
        "match-client-id": true
      }
    ]
  }
}
